controllers/volumereplicationgroup_controller.go

```diff
@@ -146,6 +156,68 @@ func (r *VolumeReplicationGroupReconciler) configMapFun(configmap client.Object)
 	return req
 }
 
+func (r *VolumeReplicationGroupReconciler) waitForCrds() error {
+	requiredCrds := []string{"VolumeReplicationClass", "VolumeReplication"}
+
+	for _, crd := range requiredCrds {
+		err := r.waitForResource(crd)
+		if err != nil {
+			r.Log.Error(err, "unexpected error occurred while checking CRD", "CRD", crd)
+
```

Comment:
No need to add blank line before the return.

Reply:
Without a blank line lint fails with error: "return with no blank line before"

Reply:
ok, we need to fix the linter, it is not helpful when the linter enforce bad style.